source/_integrations/application_credentials.markdown

```diff
@@ -10,29 +10,45 @@ ha_codeowners:
 
 ## Automatic setup
 
-Some integrations like [Google Calendar](/integrations/google/) will automatically create application credentials as part of adding an integration. To view any entries:
-1. Go to **{% my integrations title="Settings -> Devices & Services" %}**
-2. Click the overflow menu on the top right **...** and choose **Application Credentials**
+Some integrations like [Google Calendar](/integrations/google/) will automatically create application credentials as part of adding an integration.
 
-![Devices and services overflow menu](/images/integrations/application_credentials/devices-and-services-menu.png)
+## To view stored application credentials
 
-![Application credential list](/images/integrations/application_credentials/application-credentials.png)
+1. Go to {% my integrations title="**Settings** > **Devices & services**" %}.
+2. In the top right corner, select the three dots menu and select **Application credentials**.
+
+    ![Devices and services overflow menu](/images/integrations/application_credentials/devices-and-services-menu.png)
+
+    ![Application credential list](/images/integrations/application_credentials/application-credentials.png)
 
 ## Manual setup
 
-Note that integrations that do not support cloud linking will prompt to setup Application Credentials
+Some integrations do not support cloud linking. They will prompt you to set up Application Credentials
 when setting up for the first time. You may also manually enter credentials with the following steps:
 
 1. Obtain an OAuth *Client ID* and *Client Secret* from the cloud provider. The integration should have specific instructions for this.
-2. In Home Assistant navigate to **{% my integrations title="Settings -> Devices & Services" %}**
-3. Click the overflow menu on the top right **...** and choose **Application Credentials**
+2. In Home Assistant, go to {% my integrations title="**Settings** > **Devices & services**" %}.
+3. In the top right corner, select the three dots menu and select **Application credentials**.
 4. Select the integration, enter a *Name* and your OAuth *Client ID* and *Client Secret*.
-5. Save the credentials
+5. Save the credentials.
+
+    ![Manual setup](/images/integrations/application_credentials/application-credential-setup.png)
+
+6. You may then visit **Integrations** and set up the integration.
+
+## Deleting application credentials
+
+To delete an application credential, for example because you created a new one, follow these steps:
+
+1. Go to {% my integrations title="**Settings** > **Devices & services**" %}.
+
+    ![Devices and services overflow menu](/images/integrations/application_credentials/devices-and-services-menu.png)
 
-![Manual setup](/images/integrations/application_credentials/application-credential-setup.png)
+2. In the top right corner, select the three dots menu and select **Application credentials**.
+3. Select the credential from the list, select the three-dots menu and select **Delete**.
```

Comment:
The overflow menu has been added in 2024.7, so we should change the target branch to `next` in this case.